docs/docs/use_cases/tool_use/agents.ipynb

```diff
@@ -17,13 +17,13 @@
     "jp-MarkdownHeadingCollapsed": true
    },
    "source": [
-    "## Agents\n",
+    "## Repeated tool use with agents\n",
```

Comment:
Maybe a nit but repeated tool use is only one element of this right? There is also the decision of what tool to use, or whether to use any tools at all. I don't have a great alternative title, maybe something like "Agents and model-driven tool use"

Reply:
true, but was thinking that's handled by the "Choosing between multiple tools" section, what makes agents diff from just a routing chain is it chooses how many steps to take